### PR TITLE
Build the first 2 sections of the mobile website

### PR DESCRIPTION
**Pull request by hamzaw9**
This pull request includes the implementation of the first two sections of the mobile website. 

1. Toolbar/Header:
- Implemented the toolbar/header section of the website.

2. Headline Section:
- Added the headline section immediately following the toolbar/header.
- Included an introductory text that explains my work, emphasizing its linter-free and documented nature.
- Ensured a professional presentation of my work.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+    <title>Portfolio</title>
+  </head>
+  <body>
+    <div class="wrapper">
+      <!-- Start: Navbar-->
+      <nav>
+        <a class="logo" href="#">My Logo</a>
+        <i class="fas fa-bars"></i>
+      </nav>
+      <!-- End: Navbar-->
```
Comment (topeogunleye) [inline index.html]: - [ ] The project requirements specify that the container of the Toolbar section should be a header tag. However, you have used a div tag instead. [Please update the code to use a header tag instead."](https://stackoverflow.com/questions/19748487/semantic-markup-why-header-instead-of-div-id-header)[1](https://stackoverflow.com/questions/19748487/semantic-markup-why-header-instead-of-div-id-header)[2](https://discuss.codecademy.com/t/why-use-div-id-header-instead-of-header-should-we-use-html5/62596)

I hope that helps!

File: index.html [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+      <!-- Start: Main Section-->
```
Comment (topeogunleye) [inline index.html:26]: - [ ] I noticed while reviewing your code that the spacing on the horizontal axis between your Toolbar and Header is too much when compared to the Figma design.

| Your Design  | Figma Design |
| ------------- | ------------- |
| image  | image  |

**Review by topeogunleye — CHANGES_REQUESTED** [review]
Hi @hamzaw9, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

Try-Again

> ## (Highlights) Good Points: 💪 

Great job! 👏👏👏 Your project fulfills almost all the requirements for the Toolbar and Headline sections. I especially like how you used **flexbox** to align the content. 😍😍😍

Here's a detailed breakdown of your project:

Toolbar:
- The logo (if the design has one) is on the left and has a link (a tag).
- The hamburger button is on the right, and has a nav tag.
- The section has the proper background image (if the design has one).
- The paragraph (e.g., "I'm a software ...") is a p tag.
- The list of icons uses a ul tag to group them, and li per icon; each icon has a link (a tag).
- The content is aligned with flexbox.
- The section has the margins (top, bottom, left, right) defined in the design.
- The section uses the font and colors (background and font) defined for the template.

Headline:
- The container of the section is a section tag.
- The section has the proper background image (decorations are always background images, not img tags).
- The main title (e.g., "I'm Marianne ...") is a h1 tag.
- The subtitle (if existing) uses a header tag hx (examples: ```<h1>,<h2>,<h3>```, etc...).
- The paragraph (e.g., "I'm a software ...") is a `p` tag.
- The list of icons uses a ul tag to group them, and li per icon; each icon has a link (a tag).
- The content is aligned with flexbox.
- The section has the margins (top, bottom, left, right) defined in the design.
- The section uses the font and colors (background and font) defined for the template.


Keep up the good work! 😊😊😊

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- For future pull requests, please notice that your branch name `mobile-portfolio-setup` does not follow the convention of being named after a feature or action. For example, you could name your branch `added-mobile-portfolio-setup`. This makes it easier to track the changes and the purpose of each branch.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by LionRouge1 — APPROVED** [review]
Hi @hamzaw9, 

# STATUS: APPROVED! 🥇 💯 
Great work on making the changes 👏 
You've done well implementing the requested changes

done

Your project is complete! There is nothing else to say other than... it's time to merge it :shipit:

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**